### PR TITLE
[ci2] watched_branches is not a URL

### PR DESCRIPTION
**Pull request by danking**
Noticed this today.

### CODE REVIEW

File: hail/python/test/hail/expr/test_expr.py [hunk 1]
```diff
@@ -2648,7 +2648,7 @@ def test_ndarray_ref(self):
         self.assertEqual(hl.eval(h_np_cube[0, 0, 1]), 1)
         self.assertEqual(hl.eval(h_np_cube[1, 1, 0]), 6)
 
-        self.assertRaises(ValueError, hl._ndarray, [[4], [1, 2, 3], 5])
+        self.assertRaises(AttributeError, hl._ndarray, [[4], [1, 2, 3], 5])
```
Comment (cseed) [inline hail/python/test/hail/expr/test_expr.py]: What's this about?

Comment (danking) [reply hail/python/test/hail/expr/test_expr.py]: whoops

**Comment by cseed** [comment]
Failed due to intermittent dataproc failure (sign), this just needs a bump.

**Comment by cseed** [comment]
I think this will go in if you push an empty commit.

**Pull request merged**